consensus/consensus_service.go

```diff
@@ -172,9 +172,9 @@ func (consensus *Consensus) UpdatePublicKeys(pubKeys []*bls.PublicKey) int {
 	consensus.pubKeyLock.Lock()
 	consensus.PublicKeys = append(pubKeys[:0:0], pubKeys...)
 	consensus.CommitteePublicKeys = map[string]bool{}
-	utils.Logger().Info().Msg("My Committee")
-	for _, pubKey := range consensus.PublicKeys {
-		utils.Logger().Info().Str("BlsPubKey", pubKey.SerializeToHexStr()).Msg("Member")
+	utils.Logger().Info().Msg("My Committee updated")
+	for i, pubKey := range consensus.PublicKeys {
+		utils.Logger().Info().Int("index", i).Str("BlsPubKey", pubKey.SerializeToHexStr()).Msg("Member")
```

Comment:
is it info or debug?

Reply:
It's Info, since this happened only every epoch, it's importance information on its own